src/main/java/org/prebid/server/cache/CacheService.java

```diff
@@ -379,12 +380,18 @@ private static PutObject createPutObjectVideoOnly(Bid bid) {
     }
 
     /**
-     * Makes JSON type {@link PutObject} from {@link com.iab.openrtb.response.Bid}. Used for OpenRTB auction request.
+     * Makes JSON type {@link PutObject} from {@link com.iab.openrtb.response.Bid}. Used for OpenRTB auction request and
+     * add winn url.
      */
-    private PutObject createJsonPutObjectOpenrtb(CacheBid cacheBid) {
+    private PutObject createJsonPutObjectOpenrtb(CacheBid cacheBid, String accountId) {
+        final String bidId = cacheBid.getBid().getId();
```

Comment:
Pls extract `cacheBid.getBid()` to separate variable.
Then `bidId` declaration can be omitted.